### PR TITLE
chore (refs T33323): add storybook for DpSelect component.

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T33318

- add storybook for DpSelect component.

### CODE REVIEW

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  title="Components/Select"
+  component={DpSelect} />
+
+# DpSelect
+
+The DpSelect component provides `<select>` element to create a drop-down list.
+
+With `label` attribute can select menu title be passed, with `options` attribute can call a computed.
+
+And with `@select` event can call a method.
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:14]:
```suggestion
The DpSelect component is used to render a select input field with customizable options. It provides functionality for selecting an option from a dropdown list and emits a select event with the selected value.
```

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Default usage
+
+```html
+<dp-select
+  v-model="selectModel"
+  class="u-mb-0_5"
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:21]: the class does not add value in the example.

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  label="Select Title"
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:22]: the `label` prop is not a plain string but an object. See DpLabel, and DpSelect implementation.

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  name="select_menu_name"
+  :options="Computed Name"
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:24]: Here the example should show some real options in an object. see implementation.

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  required
+  @select="Method Name" />
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:26]:
```suggestion
  @select="updateLocalState" />
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: src/components/DpSelect/DpSelect.stories.mdx (hunk 2 above)
Comment (ahmad-demos) [reply src/components/DpSelect/DpSelect.stories.mdx]: Done in https://github.com/demos-europe/demosplan-ui/pull/350/commits/fcd7516cb6e2df5ee35e745788547fcfc2a1d57c

File: src/components/DpSelect/DpSelect.stories.mdx (hunk 3 above)
Comment (ahmad-demos) [reply src/components/DpSelect/DpSelect.stories.mdx]: Done in https://github.com/demos-europe/demosplan-ui/pull/350/commits/fcd7516cb6e2df5ee35e745788547fcfc2a1d57c

File: src/components/DpSelect/DpSelect.stories.mdx (hunk 4 above)
Comment (ahmad-demos) [reply src/components/DpSelect/DpSelect.stories.mdx]: Done in https://github.com/demos-europe/demosplan-ui/pull/350/commits/fcd7516cb6e2df5ee35e745788547fcfc2a1d57c

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+            id: 'selectId',
+            label: {
+              text: 'select label'
+            },
+            classes: 'layout__item u-6-of-12 sbdocs-preview css-tmvzag'
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:46]: There is only one usage of `classes` within demosplan-core. As this is an ugly opportunity to reach into the internals of the component, i'd suggest to not document it here, rather add the following in line 61 of DpSelect:

```
    /**
     * @deprecated
     */
```

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+<dp-select
+  v-model="selectModel"
+  :label="{
+          text: Translator.trans('Text')
+        }"
+  name="select_menu_name"
+  :options="selectOptions"
+  required
+  @select="updateLocalState" />
+```
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:24]: We do not need to document the html form, if we use the Storybook component implementation.

```suggestion

## Examples
```

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+                { label: 'Option 2', value: '2' }
+            ]
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+
+### Selected
+
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:104]:
```suggestion
### Selected

Pass the value of the option that should be selected.

```

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+        args={{
+            id: 'selectId',
+            label: {
+              text: 'select label'
+            },
+            options: [
+                { label: 'Option 1', value: '1' },
+                { label: 'Option 2', value: '2' }
+            ],
+            selected: 'selected'
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:117]:
```suggestion
            selected: '2'
```

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+            },
+            required: true
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+### Options
+
+Need label and value
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:83]:
```suggestion
Options are passed as an array of objects, which must have `label` and `value` properties.
```

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+
+### Disabled
+
+<Canvas>
+    <Story
+        name="Disabled"
+        args={{
+            id: 'selectId',
+            label: {
+              text: 'select label'
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:55]: For all examples: i think this label is a better fit, especially for the options below.

```suggestion
              text: 'Chose a number'
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: src/components/DpSelect/DpSelect.stories.mdx (hunk 6 above)
Comment (ahmad-demos) [reply src/components/DpSelect/DpSelect.stories.mdx]: Done in https://github.com/demos-europe/demosplan-ui/pull/350/commits/1bbcb01fa0d35f8ef47d5093c513a3ca3e69604f

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+
+If a select is `required`, an asterisk will be added to its label.
+
+<Canvas>
+    <Story
+        name="Required"
+        args={{
+            id: 'selectId',
+            label: {
+              text: 'select label'
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:46]:
```suggestion
              text: 'Choose a number'
```

File: src/components/DpSelect/DpSelect.stories.mdx [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+
+Options are passed as an array of objects, which must have `label` and `value` properties.
+
+<Canvas>
+    <Story
+        name="Options"
+        args={{
+            id: 'selectId',
+            label: {
+              text: 'select label'
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.stories.mdx:64]:
```suggestion
              text: 'Choose a number'
```

**Review by spiess-demos — APPROVED** [review]
Lgtm - thanks!

**Pull request merged**